admin_manual/configuration/files/external_storage/google.rst

```diff
@@ -14,97 +14,135 @@ If you already have a Google account, such as Groups, Drive, or Mail, you can
 use your existing login to log into the Google Cloud Console. After logging in 
 click  the **Create Project** button.
 
-.. figure:: images/google-drive.png
-   :alt: Google Drive app "create project" screen
+.. figure:: images/google_drive/001.png
+   :alt: Google Cloud Console
+
+.. figure:: images/google_drive/002
+   :alt: Create Project
+
 
 Give your project a name, and either accept the default **Project ID** or 
-create 
-your own, then click the **Create** button.
+create your own, then click the **Create** button. For this example a random name was choosen "owncloud-04-27". Feel free to choose your own name.
```

Comment:
s/For this example a random name was choosen "owncloud-04-27". Feel free to choose your own name./For this example a random name was chosen, "owncloud-04-27". However, feel free to choose your own name./g